### PR TITLE
[13.0] [MIG] mrp_production_request: Module migration in v13.

### PR DESCRIPTION
**Pull request by Chandresh-SerpentCS**
Supersedes #437

### CODE REVIEW

**Comment by Chandresh-SerpentCS** [comment]
@dreispt @bodedra Please review the PR.

**Review by bodedra — CHANGES_REQUESTED** [review]
@Chandresh-SerpentCS I followed readme information and did following configuration.

- Product A - Route - Manufacturing - Create BOM and assign 2 components
- Product A - Inventory tab - Checked Manufacturing Request

Afterwards, create a MR with Product A and create MO with compute lines.

Verified MO, it doesn't have components. 

Firefox_Screenshot_2020-04-03T12-35-53 894Z

Same thing I checked with V12.0 and works fine.

Firefox_Screenshot_2020-04-03T12-35-04 946Z

Please correct me.

CC  @LoisRForgeFlow @dreispt @AaronHForgeFlow

**Comment by Chandresh-SerpentCS** [comment]
@bodedra Request you to test the module through the new code.

**Review by dreispt — APPROVED** [review]
Code review only, LGTM

File: mrp_production_request/wizards/mrp_production_request_create_mo.py [hunk 1]
```diff
@@ -121,6 +122,11 @@ def create_mo(self):
         self.ensure_one()
         vals = self._prepare_manufacturing_order()
         mo = self.env["mrp.production"].create(vals)
+        move = mo._get_moves_raw_values()
+        component_ids = []
+        for rec in move:
+            component_ids.append((0, 0, rec))
+        mo.write({"move_raw_ids": component_ids})
```
Comment (LoisRForgeFlow) [inline mrp_production_request/wizards/mrp_production_request_create_mo.py:129]: I think you only have to confirm it, I'm v13 they added back the state "draft".

Comment (Chandresh-SerpentCS) [reply mrp_production_request/wizards/mrp_production_request_create_mo.py:129]: I just wanted to do same as v12 flow. Yes, In v13, Draft state is new added in MRP Production.

Comment (LoisRForgeFlow) [reply mrp_production_request/wizards/mrp_production_request_create_mo.py:129]: Yes, I think this is fine. I am checking and from in draft state you already have raw moves. So your fix is correct.

Can you squash this fix with the previous commit?

Comment (Chandresh-SerpentCS) [reply mrp_production_request/wizards/mrp_production_request_create_mo.py:129]: Sure.

**Comment by bodedra** [comment]
@Chandresh-SerpentCS @LoisRForgeFlow Do you know why runbot says to force rebuild? I have force rebuild multiple times but when I revisit, it sames.

Assign component to MO, code changes are fine to me.

**Comment by LoisRForgeFlow** [comment]
@bodedra I don't know, lately it has been impossible to test things out in runbot to the point I'm now always trying locally by default...

**Review by LoisRForgeFlow — APPROVED** [review]

**Review by bodedra — APPROVED** [review]
Works as expected. :+1:

File: mrp_production_request/wizards/mrp_production_request_create_mo.py (hunk 1 above)
Comment (Chandresh-SerpentCS) [reply mrp_production_request/wizards/mrp_production_request_create_mo.py:129]: > Yes, I think this is fine. I am checking and from in draft state you already have raw moves. So your fix is correct.
> 
> Can you squash this fix with the previous commit?

Squash done !

**Pull request closed without merging**